content/member/camden.md

```diff
@@ -0,0 +1,42 @@
++++
+bio = ""
+date = "2023-06-15"
+id = "camden"
+interests = ["Systems Design", "Autograder", "XAI"]
+name = "Camden Beard"
+portrait = "/portraits/camden.jpg"
+short_bio = "Camden is a 4th year Computer Science student at UCSC"
+short_name = "Camden"
```

Comment:
to change the ordering; you can add:
`sort_position=6`

Reply:
Awesome!  Fixed!